### DISCUSSION TITLE
Overdispersed Counts

### DISCUSSION THREAD

**Discussion by tmeeha** [lindeloev/mcp#163] [Q&A]
Jonas, this is a great package! Community, I have a few questions about using the package.

1) I am working with overdispersed count data. Is it possible to specify a negative binomial distribution for the response?

2) If not, one solution to dealing with overdispersion is to have an observation level random intercept. If my simple model is: 
model1 = list(
  count ~ 1 + year_c,   
  ~ 0 + year_c
),
can you give me a hint as to how to add an observation level random effect? I have a small dataset (n=20) so hopefully this isn't too computationally challenging.

3) I am looking forward to v0.04 when we can add a global covariate to the formula list. Right now I get an error when I add a second covariate to any part.

Thanks again Jonas for the great package and to the community for help. 

Tim

**Comment by lindeloev** [comment]
v0.5 should include custom families and probably a larger set af default families (v0.4 will include most of the infrastructure work to enable it). But as things stand now, I can't give an ETA since I prioritize another project higher.

One solution is to look at the fit$jags_code and make the changes necessary to do NB and/or random effects. But all the `mcpfit`-specific convenience functions of mcp (plotting, summaries, etc.) would probably not work out-of-the-box with these.
